build-system/pr-check.js

```diff
@@ -126,7 +126,9 @@ function isBuildSystemFile(filePath) {
       !isFlagConfig(filePath) &&
       // Exclude visual diff files from build-system since we want it to trigger
       // visual diff tests.
-      !isVisualDiffFile(filePath);
+      !isVisualDiffFile(filePath))
+      // OWNERS.yaml files should trigger build system to run tests
+      || path.basename(filePath) === 'OWNERS.yaml';
```

Comment:
Specifically which node tests did you want to trigger? Right now, I believe we trigger `gulp ava` for `BUILD_SYSTEM`.

Also, this is the first individual root-level file that we're considering to be a part of `BUILD_SYSTEM`. Currently, files like `.travis.yml` fall into the default bucket of `RUNTIME`. Perhaps we should pull those in too?